src/main/scala/com/linkedin/feathr/offline/anchored/keyExtractor/MVELSourceKeyExtractor.scala

```diff
@@ -86,5 +86,6 @@ private[feathr] class MVELSourceKeyExtractor(val anchorExtractorV1: AnchorExtrac
   // this helps to reduce the number of joins
   // to the observation data
   // The default toString does not work, because toString of each object have different values
-  override def toString: String = getClass.getSimpleName + " with keyExprs:" + keyExprs.mkString(" key:")
+  override def toString: String = getClass.getSimpleName + " with keyExprs:" + keyExprs.mkString(" key:") +
+    anchorExtractorV1.toString
```

Comment:
This change is a bit hard to understand. Why would we just append the anchorExtractorV1.toString to the end like this?

Reply:
At a minimum, should the toString be modified to make the output more legible? E.g. adding `"anchorExtractor: "` to the string before `anchorExtractorV1.toString`?